### ISSUE TITLE
Question about inference.

### ISSUE THREAD

**Issue by kingofengineer** [MIC-DKFZ/nnUNet#1522]
Hello!
Does NNUNet need to perform the same sum processing on the test set as the training set when inference on the test set?

**Comment by constantinulrich** [comment]
Hi, sorry for the delay we just assigned some team members to work on all issues.
Could you explain what processing of which data do you mean?
